cf/env/env.go

```diff
@@ -29,20 +31,31 @@ func New(delegate server.Environment) server.Environment {
 }
 
 type cfEnvironment struct {
-	cfEnv    *cfenv.App
+	cfEnv *cfenv.App
 	server.Environment
 }
 
-func (e *cfEnvironment) Load() error {
-	var err error
+func (e *cfEnvironment) Load() (err error) {
 	if err = e.Environment.Load(); err != nil {
-		return err
+		return
+	}
+	var postgreServiceName string
+	if serviceName := e.Environment.Get("db.name"); serviceName != nil {
+		postgreServiceName = serviceName.(string)
+	} else {
+		logrus.Warning("No PostgreSQL service name found")
+		return
 	}
 	if e.cfEnv, err = cfenv.Current(); err != nil {
-		return err
+		return
+	}
+	service, err := e.cfEnv.Services.WithName(postgreServiceName)
+	if err != nil {
+		logrus.Debug(err)
+		return fmt.Errorf("Could not find service with name %s", postgreServiceName)
```

Comment:
why not wrap the actual error that happened in the utility message so we keep some sort of trace of the error